bindings/matrix-sdk-ffi/src/room_list.rs

```diff
@@ -140,6 +140,22 @@ impl RoomListService {
         })))
     }
 
+    async fn apply_input(&self, input: RoomListInput) -> Result<(), RoomListError> {
+        self.inner.apply_input(input.into()).await.map_err(Into::into)
+    }
```

Comment:
This one should be in the other impl block too (even if it happens to work without the runtime compat code right now).